### PR TITLE
Add missing log entry tests, change method signature of train enter/l…

### LINKED ISSUE #327: Add missing log entry test for train enter/leave block section
## Acceptance Criteria
- [x] add log entry tests for train leave block section & train enter block section
- [x] change block section length to int

### PR DESCRIPTION
**Pull request by LucasDerReisende**
…eave block section block_section_length

Fixes #327 

Added tests that were missing of log entry, fixed wrong type in method signature 

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [x] Breaking changes anounced
- [x] Added classes that inherit from `BaseModel` to `src.constants.tables`
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

File: tests/logger/test_log_entry.py [hunk 1]
```diff
@@ -1101,601 +1103,946 @@ def test_deserialization_empty_fails(self, empty_set_signal_log_entry_as_dict):
...
 
-        def test_serialization(self, inject_fault_log_entry_as_dict):
-            """Test that Inject Fault Log Entry can be serialized."""
-            inject_fault_log_entry = InjectFaultLogEntry.create(
-                **inject_fault_log_entry_as_dict
+        def test_serialization(self, train_enter_block_section_log_entry_as_dict):
+            """Test that TrainEnterBlockSectionLogEntry can be serialized."""
+            train_enter_block_section_log_entry = TrainEnterBlockSectionLogEntry.create(
+                **train_enter_block_section_log_entry_as_dict
             )
```
Comment (instant-sky) [inline tests/logger/test_log_entry.py]: @Lietze and @CR1337 told me yesterday for a similar test of me, that for testing the serialization a `to_dict` is missing. What is correct now?

Comment (instant-sky) [reply tests/logger/test_log_entry.py]: nervermind, you test that below :)

File: tests/logger/test_log_entry.py [hunk 2]
```diff
@@ -1101,601 +1103,946 @@ def test_deserialization_empty_fails(self, empty_set_signal_log_entry_as_dict):
...
+        self,
+        timestamp,
+        tick,
+        message,
+        run,
+        platform_blocked_fault_configuration,
+        track_blocked_fault_configuration,
+        track_speed_limit_fault_configuration,
+        schedule_blocked_fault_configuration,
+        train_prio_fault_configuration,
```
Comment (instant-sky) [inline tests/logger/test_log_entry.py]: I guess everything from here on are the parts that are shown as removed everywhere above?

**Review by instant-sky — COMMENTED** [review]
Nice feature!

File: tests/logger/test_log_entry.py (hunk 2 above)
Comment (LucasDerReisende) [reply tests/logger/test_log_entry.py]: Please have a look at it again, I changed the identation of other classes, I accidentally changed

**Review by instant-sky — APPROVED** [review]
Thanks for clarifying! Looks good now

**Pull request merged**